app/models/piece.rb

```diff
@@ -1,5 +1,26 @@
 class Piece < ApplicationRecord
   belongs_to :game
+  validates :color, presence: true
+  validates :type, presence: true
+
+  scope :white, ->() { where(color: 'white') }
+  scope :black, ->() { where(color: 'black') }
+  #
+  # def self.at(x_position, y_position)
+  #   Piece.find_by(x_position: x_position, y_position: y_position)
+  # end
```

Comment:
Can delete commented out code. No need to keep it around. 😄

Reply:
Thank Mike. I will remove it when I work on the next task.